src/wound.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
```

Comment:
This is missing the traditional style of header guards that most other headers have.